Adding explanation for combining until with loop (#1367)
ansible/ansible-documentation: docs/docsite/rst/playbook_guide/playbooks_loops.rst

If ``until`` is not specified, the task will retry until the task succeeds but at most ``retries`` times.

> You can combine ``until`` with ``loop`` or ``with_<lookup>``, the result of the task for each element of the loop will be registered in the variable and can be used in the ``until`` condition. Here's an example:

Reviewer: ```suggestion
You can combine the ``until`` keyword with ``loop`` or ``with_<lookup>``. The result of the task for each element of the loop is registered in the variable and can be used in the ``until`` condition. Here is an example:
```

.. code-block:: yaml

    - name: Retry combined with a loop
      uri:
        url: "https://{{ item }}.ansible.com"
        method: GET
      register: uri_output
      with_items:

>       - "abc"

Reviewer: ```suggestion
      - "galaxy"
```

      - "docs"

>       - "xyz"

Reviewer: ```suggestion
      - "forum"
```
